php/README.md

```diff
@@ -0,0 +1,136 @@
+# PHP
+
+## PSR-2
+
+To increase the readability of all the code, it is a good idea to go towards a standard that we all conform to. The PHP world has developed a nice standard for this: [The PSR-2 standard](https://www.php-fig.org/psr/psr-2). For the most used IDEs I will post a link here for how to set this up:
+
+[PHPStorm code sniffer](https://confluence.jetbrains.com/display/PhpStorm/PHP+Code+Sniffer+in+PhpStorm)
+
+## SOLID
```

Comment:
Write headings as simple convention rules/actions. For example: "Use SOLID when designing your application"
Also explain what 'SOLID' is and means.